addon/helpers/ember-sparkles/axis.js

```diff
@@ -10,8 +10,13 @@ export function emberSparklesAxis([ scale ], { position, tickFormat, ticks, widt
 
   let result = axisFn().scale(scale);
 
-  if (tickFormat) {
-    result.tickFormat(timeFormat(tickFormat));
+  switch(typeof tickFormat) {
```

Comment:
this appears to do the same thing regardless of the input type? why does it need a `switch` ?